src/averageAges.js

```diff
@@ -37,7 +51,37 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(
+    person => person['sex'] === 'f'
+  );
+
+  let womenWithCheldren = women;
+
+  if (withChildren === true) {
```

Comment:
```suggestion
  if (withChildren) {
```
also better to make ternary inside filter callback